src/appleseed/foundation/math/scalar.h

```diff
@@ -380,16 +384,25 @@ inline T pow_int(const T x)
 }
 
 template <typename T>
-inline T pow_int(const T x, size_t p)
+inline T pow_int(T x, size_t p)
 {
     // todo: implement exponentiation by squaring.
```

Comment:
Remove todo note now that it's done (but keep the reference).

Same thing in the other function.